content/presence-occupancy/presence.textile

```diff
@@ -590,6 +591,28 @@ if page.hasNext() {
 }
 ```
 
+h3(#synced). Synced presence set
+
+A common need is to keep an updated list of members that are currently present on a channel in your user interface.
+
+Many developers try to build the initial list using the "@get()@":/api/realtime-sdk/presence#get method and then mutate that list whenever a new presence event arrives. This approach is not recommended by Ably as it's easy to get wrong and end up with a list that's out of sync with the real presence set.
```

Comment:
Possible reword for clarity? 

Ably advises against this method due to its high risk of errors, which could lead to discrepancies between your list and the actual presence set.